rosbridge_library/src/rosbridge_library/capabilities/defragmentation.py

```diff
@@ -87,17 +87,17 @@ def defragment(self, message):
                 time_diff = now - self.received_fragments[id]["timestamp_last_append"]
                 if (time_diff.total_seconds() > self.fragment_timeout and
                     not self.received_fragments[id]["is_reconstructing"]):
-                    log_msg = "fragment list " + str(id) + " timed out.."
+                    log_msg = ["fragment list " + str(id) + " timed out.."]
 
                     if message["id"] != id:
-                        log_msg += " -> removing it.."
+                        log_msg.extend([" -> removing it.."])
```

Comment:
When you're just adding a single element, like here, a simple `append()` (without the square brackets) is enough